<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveDailog</class>
 <widget class="QWidget" name="ReceiveDailog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>22</number>
   </property>
   <property name="leftMargin">
    <number>31</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>27</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>22</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_information">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>209</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>209</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #ffffff;</string>
           </property>
           <property name="text">
            <string>PAYMENT INFORMATION</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>61</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label_uri">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>37</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>37</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #ffffff;</string>
           </property>
           <property name="text">
            <string>URI: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_uri_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #ffdfb5;</string>
           </property>
           <property name="text">
            <string>dogecash:DF4zEBdZeG5UyQvmz4a9LDk4w6QrHMdBa4</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>14</number>
         </property>
         <item>
          <widget class="QLabel" name="label_address">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>84</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>84</width>
             <height>81</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #ffffff;</string>
           </property>
           <property name="text">
            <string>ADDRESS:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_address_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent;
color: #ffdfb5;</string>
           </property>
           <property name="text">
            <string>DF4zEBdZeG5UyQvmz4a9LDk4w6QrHMdBa4</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>22</number>
         </property>
         <property name="topMargin">
          <number>51</number>
         </property>
         <item>
          <widget class="QPushButton" name="button_uri">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #b6a27e;
color: #f2f2f2;</string>
           </property>
           <property name="text">
            <string>COPY URI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_address">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #b6a27e;
color: #f2f2f2;</string>
           </property>
           <property name="text">
            <string>COPY ADDRESS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_remove">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #b6a27e;
color: #f2f2f2;</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>585</width>
           <height>104</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
